source/module_hsolver/kernels/cuda/math_kernel_op.cu

```diff
@@ -10,9 +10,9 @@
 #include <thrust/execution_policy.h>
 #include <thrust/inner_product.h>
 
-#define WARP_SIZE 32
-#define FULL_MASK 0xffffffff
-#define THREAD_PER_BLOCK 256
+const int WARP_SIZE = 32;
```

Comment:
In this PR, you have changed the macro to const variables, the name of variables should be lower-case instead of upper-case.